opentelemetry-otlp/tests/integration_test/Cargo.toml

```diff
@@ -9,11 +9,13 @@ publish = false
 once_cell = { workspace = true }
 opentelemetry = { path = "../../../opentelemetry", features = ["metrics", "logs"] }
 opentelemetry_sdk = { path = "../../../opentelemetry-sdk", features = ["rt-tokio", "logs", "testing"] }
-opentelemetry-otlp = { path = "../../../opentelemetry-otlp", features = ["tonic", "metrics", "logs"] }
-opentelemetry-semantic-conventions = { path = "../../../opentelemetry-semantic-conventions" }
-opentelemetry-proto = { path = "../../../opentelemetry-proto", features = ["gen-tonic-messages", "trace", "with-serde"] }
-opentelemetry-appender-log = { path = "../../../opentelemetry-appender-log", default-features = false}
+opentelemetry-proto = { path = "../../../opentelemetry-proto", features = ["gen-tonic-messages", "trace", "logs", "with-serde"] }
 log = { workspace = true }
 tokio = { version = "1.0", features = ["full"] }
 serde_json = "1"
 testcontainers = "0.15.0"
+
+[target.'cfg(unix)'.dependencies]
+opentelemetry-appender-log = { path = "../../../opentelemetry-appender-log", default-features = false}
+opentelemetry-otlp = { path = "../../../opentelemetry-otlp", features = ["tonic", "metrics", "logs"] }
+opentelemetry-semantic-conventions = { path = "../../../opentelemetry-semantic-conventions" }
```

Comment:
why are these dependencies unix specific?

Reply:
The integration tests are only run for logs: https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-otlp/tests/integration_test/tests/logs.rs#L1